string io_var
bool io_value
---
# Success if empty
string error
